src/components/PopupWithConfirmation.js

```diff
@@ -0,0 +1,39 @@
+// child class of Popup
+
+import Popup from "./Popup.js";
+export default class PopupWithConfirmation extends Popup {
+  constructor(popupSelector, handleFormSubmit, confirmationButtonSelector) {
+    console.log(popupSelector);
+    super(popupSelector);
+    this._handleFormSubmit = handleFormSubmit;
+    this._confirmationButton = this.popup.querySelector(
+      confirmationButtonSelector
+    );
+  }
+
+  setId(id) {
+    this._id = id;
+  }
+
+  getId() {
+    return this._id;
+  }
+
+  setCard(card) {
+    this._card = card;
+  }
+
+  removeCard() {
```

Comment:
you should remove the card element in Card class, since it's responsible for the card rendering